go.mod

```diff
@@ -17,15 +17,15 @@ require (
 	github.com/chzyer/readline v1.5.1
 	github.com/cockroachdb/apd/v2 v2.0.2
 	github.com/cometbft/cometbft v0.37.2
-	github.com/cometbft/cometbft-db v0.7.0
+	github.com/cometbft/cometbft-db v0.8.0
```

Comment:
Mmmh: https://github.com/cometbft/cometbft-db/blob/main/CHANGELOG.md#v080
I do not like this bump.

Reply:
is downgrading possible?

Reply:
Yes, it's dependabot bumping other things in the PR. Just downgraded.